src/views/network/IngressEgressRuleConfigure.vue

```diff
@@ -79,9 +79,9 @@
       size="small"
       style="overflow-y: auto"
       :columns="columns"
-      :dataSource="rules"
+      :dataSource="rules.slice(pagesize*(page - 1), Math.min( rules.length, pagesize*page))"
```

Comment:
@davidjumani fix parenthesis/space issue

Reply:
Does the API not support pagination? Otherwise, I would expect to simply give all the datasource (not a slice) and instead use table's own pagination than an explicit one.

Reply:
It calls the listSecurityGroups api and that in turn returns all the rules associated with the SG in the response, so paginating it over there